components/GlobalStyle/sub-components/Typography.jsx

```diff
@@ -90,9 +90,17 @@ ul, ol {
 const Typography = () => (
   <>
     <Style />
+    <link
+      crossOrigin="anonymous"
+      rel="preload"
+      href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap"
+      as="font"
+      type="font/woff2"
+    />
     <link
       href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap"
       rel="stylesheet"
+      as="style"
```

Comment:
https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/link#attr-as
I want you to read the documentation calmly, as I instructed you. Always look for content from great references on the internet. There are many people on the internet who use some resources erroneously.

Reply:
I checked in the documentation that the "from" attribute is only used on links that use (preload).